agent:
  state_size: 12
  action_size: 4  # Change to 4 for a larger action space
  buffer_size: 50000
  batch_size: 62
  gamma: 0.99
  lr: 1e-3

env:
  position_err: [0, 0, 0]
  v_err: [0, 0, 0]
  orientation_err: [0, 0, 0]
  omega_ref_err: [0, 0, 0]
  joint_positions: [0, 0]
  setpoint_size: 12
  stabilization_time: 2.0
  episode_time_limit: 100000

reward:
  lambda_values: [0.2, 0.0, 0.0, 1.0, 0.5, 0.1, 0.1, 0.5, 0.4, 0.2, 0.5, 1.0]
  a: 0.7
  xi: 0.02
  theta: 0.05
  zeta: 0.1
  alpha: 0.1
  small_lambda: 0.2